projects/core/src/state/utils/entity-loader/entity-loader.action.ts

```diff
@@ -78,6 +79,26 @@ export class EntityLoadAction implements EntityLoaderAction {
 export class EntityFailAction implements EntityLoaderAction {
   type = ENTITY_FAIL_ACTION;
   readonly meta: EntityLoaderMeta;
+
+  // eslint-disable-next-line @typescript-eslint/unified-signatures -- needed to deprecate only the old constructor
+  constructor(entityType: string, id: EntityId, error: ActionErrorProperty);
+  /**
+   * @deprecated Please use the `error` parameter with a non-null, non-undefined value.
+   *             Support for `null` or `undefined` will be removed in future versions,
+   *             along with the feature toggle `ssrStrictErrorHandlingForHttpAndNgrx`.
+   */
+  constructor(
+    entityType: string,
+    id: EntityId,
+    // eslint-disable-next-line @typescript-eslint/unified-signatures -- needed to deprecate only the old constructor
+    error: any
+  );
+  /**
+   * @deprecated Please use the `error` parameter with a non-null, non-undefined value.
+   *             The `error` parameter will become required in future versions,
+   *             along with removing the feature toggle `ssrStrictErrorHandlingForHttpAndNgrx`.
+   */
+  constructor(entityType: string, id: EntityId);
```

Comment:
[NOTE]

In this exceptional case I added 2 (but not 1) deprecated constructors, because:
- `error` can be `any` 
- AND MOREOVER `error` can be not passed at all (as `error?` argument is optional)